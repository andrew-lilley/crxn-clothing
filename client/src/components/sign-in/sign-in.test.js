import React from 'react';
import { shallow } from 'enzyme';
import { SignIn } from './sign-in.component';

describe('SignIn component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SignIn />);
  });

  test('should render SignIn component', () => {
    expect(wrapper).toMatchSnapshot();
  });

});