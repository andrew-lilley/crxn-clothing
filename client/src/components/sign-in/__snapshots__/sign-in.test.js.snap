// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn component should render SignIn component 1`] = `
<styled.div>
  <styled.h2>
    I already have an account
  </styled.h2>
  <span>
    Sign in with your email and password
  </span>
  <form
    onSubmit={[Function]}
  >
    <FormInput
      handleChange={[Function]}
      label="Email"
      name="email"
      required={true}
      type="email"
      value=""
    />
    <FormInput
      handleChange={[Function]}
      label="Password"
      name="password"
      required={true}
      type="password"
      value=""
    />
    <styled.div>
      <CustomButton
        type="submit"
      >
        Sign in
      </CustomButton>
      <CustomButton
        isGoogleSignIn={true}
        type="button"
      >
        Sign in with Google
      </CustomButton>
    </styled.div>
  </form>
</styled.div>
`;
